### ISSUE TITLE
Use `Readline` for getting input from user.

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/js-basics#39]
See https://nodejs.org/api/readline.html

Refactor material that uses `readline-sync` to avoid promoting poor JavaScript habits.

This requires scaffolding the asynchronous nature of aquiring input using `Readline`.

**Comment by jrhorn424** [comment]
I agree with this change, but I did not leave myself enough time to implement for this iteration.

**Comment by raq929** [comment]
Not doing it for this iteration, but Antony's idea for this is to have a function that checks every x milliseconds to see if a boolean is true or false, and then exits when that boolean changes. Ask him for details.

**Comment by gaand** [comment]
Please see https://github.com/ga-wdi-boston/js/tree/readline

A lot of the code in the files should go on the solution branch, but not `bin/guess.js`  most of that code should remain.

This branch partially addresses #51 #54 #56
